addon/components/o-s-s/text-area.ts

```diff
@@ -0,0 +1,46 @@
+import { assert } from '@ember/debug';
+import { action } from '@ember/object';
+import Component from '@glimmer/component';
+
+interface OSSTextAreaArgs {
+  rows?: number;
+  resize?: 'vertical' | 'horizontal' | 'both' | 'none';
+  value?: string;
+  disabled?: boolean;
+  errorMessage?: string;
+  placeholder?: string;
+  onChange?(value: string): void;
+}
+
+export default class OSSTextArea extends Component<OSSTextAreaArgs> {
+  constructor(owner: unknown, args: OSSTextAreaArgs) {
+    super(owner, args);
+
+    if (this.args.resize) {
+      assert(
+        '[component][OSS::TextArea] The @resize parameter should be a value of resize',
+        ['vertical', 'horizontal', 'both', 'none'].includes(this.args.resize)
+      );
+    }
+  }
+
+  get rows(): number {
+    return this.args.rows || 1;
```

Comment:
```suggestion
    return this.args.rows || 2;
```
Maybe set the same default value that define in [MDN](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/textarea) for row